modules/azure/azurite/options.go

```diff
@@ -8,12 +8,51 @@ import (
 
 type options struct {
 	// EnabledServices is a list of services that should be enabled
-	EnabledServices []service
+	EnabledServices []Service
 }
 
 func defaultOptions() options {
 	return options{
-		EnabledServices: []service{blobService, queueService, tableService},
+		EnabledServices: []Service{BlobService, QueueService, TableService},
+	}
+}
+
+// Satisfy the testcontainers.ContainerCustomizer interface
+var _ testcontainers.ContainerCustomizer = (Option)(nil)
+
+// Option is an option for the Azurite container.
+type Option func(*options) error
+
+// Customize is a NOOP. It's defined to satisfy the testcontainers.ContainerCustomizer interface.
+func (o Option) Customize(*testcontainers.GenericContainerRequest) error {
+	// NOOP to satisfy interface.
+	return nil
+}
+
+// WithEnabledServices is a custom option to specify which services should be enabled.
+func WithEnabledServices(services ...Service) Option {
```

Comment:
suggestion: let's add a unit test for this option, i.e. to capture the error on duplicated services.